meson.build

```diff
@@ -0,0 +1,119 @@
+project(
+  'libdicom',
+  'c',
+  license: 'MIT',
+  meson_version : '>=0.54',
+  version: '0.1.0',
+)
+
```

Comment:
How about:

```
cc = meson.get_compiler('c')

cc.has_header('stdint.h', required: true)
cc.has_header('stdlib.h', required: true)
cc.has_header('string.h', required: true)
```

Since these are not on all platforms? Or maybe with a loop.

Reply:
libdicom is documented to require at least C11 in the library and C99 in headers, and both standards appear to require all three headers.  Are there supported systems where those headers might not be available?

Reply:
Ah, I didn't realize C99 had these, that's great.